### PR TITLE
Add [`un`]`fold` icons

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

File: icons/fold.svg
Comment (karsa-mistmere) [inline icons/fold.svg:1]: These should use the same dotted line as our flip icons: https://lucide.dev/icons?search=flip

Maybe we should also add both orientations too.

File: icons/fold-vertical.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "arrow",
+    "collapse",
+    "fold",
+    "vertical",
+    "dashed"
+  ],
+  "categories": [
+    "arrows"
```
Comment (karsa-mistmere) [inline icons/fold-vertical.json:11]:
```suggestion
    "arrows",
    "layout"
```

File: icons/unfold-horizontal.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "arrow",
+    "collapse",
+    "fold",
+    "vertical",
+    "dashed"
+  ],
+  "categories": [
+    "arrows"
```
Comment (karsa-mistmere) [inline icons/unfold-horizontal.json:11]:
```suggestion
    "arrows",
    "layout"
```

File: icons/unfold-vertical.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "arrow",
+    "expand",
+    "vertical",
+    "dashed"
+  ],
+  "categories": [
+    "arrows"
```
Comment (karsa-mistmere) [inline icons/unfold-vertical.json:10]:
```suggestion
    "arrows",
    "layout"
```

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]
I think these should probably go to the `layout` category. Other than that LGTM.

File: icons/fold-horizontal.svg [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  stroke-linejoin="round"
+>
+  <polyline points="19,9 16,12 19,15" />
+  <line x1="2" x2="8" y1="12" y2="12" />
+  <path d="M12 2v2" />
+  <path d="M12 8v2" />
+  <path d="M12 14v2" />
+  <path d="M12 20v2" />
+  <line x1="22" x2="16" y1="12" y2="12" />
+  <polyline points="5,15 8,12 5,9" />
```
Comment (karsa-mistmere) [inline icons/fold-horizontal.svg:19]:
```suggestion
  <path d="M2 12h6" />
  <path d="M22 12h-6" />
  <path d="M12 2v2" />
  <path d="M12 8v2" />
  <path d="M12 14v2" />
  <path d="M12 20v2" />
  <path d="m19 9-3 3 3 3" />
  <path d="m5 15 3-3-3-3" />
```

File: icons/fold-vertical.svg [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  stroke-linejoin="round"
+>
+  <line x1="12" x2="12" y1="8" y2="2" />
+  <polyline points="15,19 12,16 9,19" />
+  <path d="M4 12H2" />
+  <path d="M10 12H8" />
+  <path d="M16 12h-2" />
+  <path d="M22 12h-2" />
+  <polyline points="15,5 12,8 9,5" />
+  <line x1="12" x2="12" y1="22" y2="16" />
```
Comment (karsa-mistmere) [inline icons/fold-vertical.svg:19]:
```suggestion
  <path d="M12 22v-6" />
  <path d="M12 8V2" />
  <path d="M4 12H2" />
  <path d="M10 12H8" />
  <path d="M16 12h-2" />
  <path d="M22 12h-2" />
  <path d="m15 19-3-3-3 3" />
  <path d="m15 5-3 3-3-3" />
```

File: icons/unfold-horizontal.svg [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  stroke-linejoin="round"
+>
+  <polyline points="5,9 2,12 5,15" />
+  <line x1="8" x2="2" y1="12" y2="12" />
+  <path d="M12 2v2" />
+  <path d="M12 8v2" />
+  <path d="M12 14v2" />
+  <path d="M12 20v2" />
+  <line x1="16" x2="22" y1="12" y2="12" />
+  <polyline points="19,15 22,12 19,9" />
```
Comment (karsa-mistmere) [inline icons/unfold-horizontal.svg:19]:
```suggestion
  <path d="M16 12h6" />
  <path d="M8 12H2" />
  <path d="M12 2v2" />
  <path d="M12 8v2" />
  <path d="M12 14v2" />
  <path d="M12 20v2" />
  <path d="m19 15 3-3-3-3" />
  <path d="m5 9-3 3 3 3" />
```

File: icons/unfold-vertical.svg [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  stroke-linejoin="round"
+>
+  <line x1="12" x2="12" y1="8" y2="2" />
+  <polyline points="15,5 12,2 9,5" />
+  <path d="M4 12H2" />
+  <path d="M10 12H8" />
+  <path d="M16 12h-2" />
+  <path d="M22 12h-2" />
+  <polyline points="15,19 12,22 9,19" />
+  <line x1="12" x2="12" y1="22" y2="16" />
```
Comment (karsa-mistmere) [inline icons/unfold-vertical.svg:19]:
```suggestion
  <path d="M12 22v-6" />
  <path d="M12 8V2" />
  <path d="M4 12H2" />
  <path d="M10 12H8" />
  <path d="M16 12h-2" />
  <path d="M22 12h-2" />
  <path d="m15 19-3 3-3-3" />
  <path d="m15 5-3-3-3 3" />
```

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

**Review by karsa-mistmere — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]

**Pull request merged**